.devops/azure-templates/receivers-commands/send-receiver-command.yml

```
@@ -0,0 +1,50 @@
+#This template will handle command sending to event dispatcher to enable/disable receivers
+parameters:
+  # deployment environment
+  - name: ENV
+    type: string
+    values:
+      - DEV
+      - UAT
+      - PROD
+  # command to be sent
+  - name: COMMAND
+    type: string
+    values:
+      - START
+      - STOP
+  # command target deployment version
+  - name: TARGET_DEPLOYMENT_VERSION
+    type: string
+    values:
+      - PROD
+      - STAGING
+
+steps:
+  - task: Bash@3
+    name: ${{ parameters.COMMAND }}_receivers_${{ parameters.TARGET_DEPLOYMENT_VERSION }}
+    displayName: 'Sending command ${{ parameters.COMMAND }} to deployment ${{ parameters.TARGET_DEPLOYMENT_VERSION }}'
+    inputs:
+      targetType: "inline"
+      script: |
+        command=${{ parameters.COMMAND }}
+        deploymentTargetVersion=${{ parameters.TARGET_DEPLOYMENT_VERSION }}
+        echo "Sending command: [$command] with deployment target: [$deploymentTargetVersion] to event dispatcher instance: [$deploymentInstance]"
```

Do you mean "transactions scheduler"?

```suggestion
        echo "Sending command: [$command] with deployment target: [$deploymentTargetVersion] to transactions scheduler instance: [$deploymentInstance]"
```

---

Resolved by https://github.com/pagopa/pagopa-ecommerce-transactions-scheduler-service/pull/162/commits/9a69ab3168af0434ccf8cd457a7ee48c3996d991